hudi-common/src/main/java/org/apache/hudi/common/util/SerializationUtils.java

```diff
@@ -121,50 +116,16 @@ Object deserialize(byte[] objectData) {
 
     public Kryo newKryo() {
 
-      Kryo kryo = new KryoBase();
+      Kryo kryo = new Kryo();
       // ensure that kryo doesn't fail if classes are not registered with kryo.
       kryo.setRegistrationRequired(false);
       // This would be used for object initialization if nothing else works out.
-      kryo.setInstantiatorStrategy(new org.objenesis.strategy.StdInstantiatorStrategy());
+      kryo.setInstantiatorStrategy(new Kryo.DefaultInstantiatorStrategy(new StdInstantiatorStrategy()));
```

Comment:
I guess, this achieves the same purpose as the code in `newInstantiator` that you deleted below?

Reply:
Very right.